app/views/consignment_references.scala.html

```diff
@@ -14,58 +14,62 @@
  * limitations under the License.
  *@
 
+  
 @import controllers.routes
 @import forms.ConsignmentReferences
 @import forms.ConsignmentReferences.AllowedReferences._
 @import models.requests.JourneyRequest
-@import uk.gov.hmrc.play.views.html._
-@import views.components.fields.RadioOption
 @import views.Title
 
-@this(main_template: views.html.main_template)
+@import uk.gov.hmrc.govukfrontend.views.html.components._
 
-@(form: Form[ConsignmentReferences])(implicit request: JourneyRequest[_], messages: Messages)
+@this(
+  govukLayout: views.html.gds_main_template,
+  govukButton: GovukButton,
+  govukRadios: GovukRadios,
+  exportsInputText: components.gds.exportsInputText,
+  errorSummary: components.errorSummary,
+  formHelper: uk.gov.hmrc.play.views.html.helpers.FormWithCSRF
+)
 
-@main_template(title = Title("consignmentReferences.title", None)) {
-    @helper.form(routes.ConsignmentReferencesController.saveConsignmentReferences(), 'autoComplete -> "off") {
-        @helper.CSRF.formField
+@(form: Form[ConsignmentReferences])(implicit request: JourneyRequest[_], messages: Messages)
 
-        @components.back_link(routes.ChoiceController.displayChoiceForm())
+@govukLayout(
+  title = Title(messages("consignmentReferences.title")),
```

Comment:
Just spotted this.   You need to pass the message key so that the title is properly formatted.